eng/common/core-templates/job/job.yml

```
@@ -24,6 +24,7 @@ parameters:
   enablePublishTestResults: false
   enablePublishUsingPipelines: false
   enableBuildRetry: false
+  installMicrobuildForMac: false
```

```suggestion
  enableMicrobuildForMac: false
```

To keep things consistent with the other parameters.